ui/tests/integration/components/page/error-test.js

```diff
@@ -55,4 +56,29 @@ module('Integration | Component | page/error', function (hooks) {
         .hasText(this.error.errors[index], 'Error detail renders');
     });
   });
+
+  test('it should handle api client errors', async function (assert) {
+    this.error = getErrorResponse();
+
+    await render(hbs`<Page::Error @error={{this.error}} />`);
+
+    assert
+      .dom('[data-test-page-error-title]')
```

Comment:
A lot of these selectors are in the GENERAL selector file
```suggestion
      .dom(GENERAL.pageError.errorTitle("404"))
```